operator/config/rbac/forklift-imageio-populator_service_account.yaml

```diff
@@ -0,0 +1,5 @@
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: imageio-populator-account
+  namespace: konveyor-forklift
```

Comment:
would it make sense to set this role also to openstack populator and to the new virt-v2v pod later on? if so, let's generalize it. and the namespace should not be `konveyor-forklift` explicitly